internal/command/command.go

```diff
@@ -129,6 +128,11 @@ func (c Command) AddToParent(parent *cobra.Command) {
 			panic("command implementation needs to provide run functionality")
 		}
 
+		err = util.TrackCommandUsage(c.Cmd)
+		if err != nil {
+			sentry.CaptureException(err)
```

Comment:
Based on https://docs.sentry.io/product/sentry-basics/integrate-backend/capturing-errors/, seems that sentry is already reporting errors automatically. But just explicitly capturing errors for command tracking just in case here.

Reply:
We need to explicitly do that since we handle all the errors in a function just below called `handleError`

Reply:
so is the sentry capture exception here good?